app/views/outcomes-report/OutcomesReportResultComponent.vue

```diff
@@ -49,10 +49,20 @@ export default Vue.extend({
       let languageStats = {}
       let totalPrograms = 0
       for (const language of ['python', 'javascript', 'cpp']) {
-        const programs = this.org.progress.programsByLanguage[language]
+        let programs
+        if (this.org.newProgress) {
+          programs = this.org.newProgress.languages[language]
+        } else {
+          programs = this.org.progress.programsByLanguage[language]
+        }
         totalPrograms += programs
         languageStats[language] = { programs }
       }
+      if (this.org.newProgress) {
+        let programs = this.org.newProgress.languages.java
```

Comment:
why hardcoding java?

Reply:
just, if add java into the array, the old outcome-report would break. so currently hardcoding it until we totally replace old one with new one